app/javascript/features/lobby/display/BattleCardPlayer.jsx

```diff
@@ -167,12 +174,14 @@ export const BattleCardPlayer = ({
           <Box className={classes.wrapperRow}>
             <TeamBalanceIcon team={teamBalance} isFull={isFull}/>
             <Typography variant="h5" color="secondary" className={classes.selfPlayerName}>{playerName}</Typography>
+            {isAdmin ? <Typography variant="h5" color="darkgrey" theme={eloTheme}
+                                 className={classes.PlayerElo}> <sup>{playerElo}</sup></Typography> : null}
```

Comment:
Can you add a little padding between the player name and the elo?

Reply:
This was somewhat intentaional, trying to make it look like superscript

Reply:
I understand, just they're almost touching. Adding maybe 0.25rem padding left on the elo typography would be enough